subsys/net/lib/nrf_cloud/Kconfig.nrf_cloud_coap

```diff
@@ -36,6 +36,13 @@ config NRF_CLOUD_COAP_GF_CONF
 	  configuration values do_reply, fallback, and hi_conf. When the cloud
 	  support is ready, NRF_CLOUD_COAP_GF_CONF can be set true.
 
+config NRF_CLOUD_COAP_KEEPOPEN
+	bool "Enable SO_KEEPOPEN to prevent unnecessary socket closures"
+	default y if (SOC_NRF9161_LACA || SOC_NRF9151_LACA || SOC_NRF9131_LACA)
```

Comment:
`depends on SOC_NRF9161_LACA || SOC_NRF9151_LACA || SOC_NRF9131_LACA`

?

Reply:
How is that better? We want people to use SO_KEEPOPEN. Or are you proposing adding your `depends` line and not replacing my `default` line?

Reply:
Does this option depend on one of those values?
Does it work for the 9160?

You can keep the default line... or maybe just default y if that doesn't cause an error for 9160 builds.

Reply:
It does not work for the 9160. Earlier in the review cycle I allowed this as a benign error, but y'all asked me to limit it to the proper devices.

I'll add `depends on`.